### PR TITLE
Improved Ui of Choose File input and added validation in upload new file form in Consultation Files Page

### LINKED ISSUE #3005: Prevent File Form from submitting without validation and improving "Choose file" UI in Consultation files page
# Bugs 

# Prevent File Form from submitting without validation
https://user-images.githubusercontent.com/59426397/177199620-382fb331-9d9e-4195-8550-48ef5aaccd71.mp4

# Improving "Choose file" UI on Consultation files page
image

### PR DESCRIPTION
**Pull request by patelaryan7751**
Fixes #3005 

# Bugs Fixed

## Added validation in upload new file form
https://user-images.githubusercontent.com/59426397/177238111-84f6004e-db05-4cf6-98ac-4adf20915e9a.mp4

## Improved Ui of Choose File input
image

### CODE REVIEW

**Review by kunatastic — APPROVED** [review]

**Review by cp-Coder — APPROVED** [review]

**Comment by nihal467** [comment]
@patelaryan7751 make the choose file button rectangle itself rather than curve in the end, since everywhere we use rectangle buttons,

**Comment by patelaryan7751** [comment]
Hey, @nihal467 is this okay
image

**Comment by nihal467** [comment]
test approved

**Comment by khavinshankar** [comment]
@patelaryan7751 making the filename mandatory can reduce the user experience, so can you make it similar to audio file, giving some template name? 

@nihal467 what do you think?

**Comment by patelaryan7751** [comment]
@khavinshankar @nihal467 i also think the same shall i implement it ?

**Comment by patelaryan7751** [comment]
@nihal467 @khavinshankar is there anything else to be done?

File: src/Components/Patient/FileUpload.tsx [hunk 1]
```diff
@@ -468,24 +469,33 @@ export const FileUpload = (props: FileUploadProps) => {
...
-  const handleUpload = async (status: any) => {
+  const validateFileUpload = () => {
+    const filenameLength = uploadFileName.trim().length;
     const f = file;
     if (f === undefined) {
-      Notification.Error({
-        msg: "Please choose a file to upload",
-      });
-      return;
+      return false;
```
Comment (khavinshankar) [inline src/Components/Patient/FileUpload.tsx:484]:
```suggestion
      setUploadFileNameError("Please choose a file to upload");
      return false;
```

File: src/Components/Patient/FileUpload.tsx [hunk 2]
```diff
@@ -765,9 +776,15 @@ export const FileUpload = (props: FileUploadProps) => {
...
-                      title="changeFile"
+                      title="Choose File"
                       onChange={onFileChange}
                       type="file"
+                      className="block w-full text-sm text-slate-500
+                      file:mr-4 file:py-2 file:px-4 
+                            file:border-0
+                      file:text-sm file:font-semibold
+                      file:bg-primary-100 file:text-primary-700
+                      hover:file:bg-primary-300"
```
Comment (khavinshankar) [inline src/Components/Patient/FileUpload.tsx]:
```suggestion
                      className="block w-full text-sm text-slate-500 file:mr-4 file:py-2 file:px-4 file:border-0 file:text-sm file:font-semibold file:bg-primary-100 file:text-primary-700 hover:file:bg-primary-300"
```

**Review by khavinshankar — CHANGES_REQUESTED** [review]

**Comment by patelaryan7751** [comment]
@khavinshankar i have done the changes

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**